code/client/scripts/index.js

```diff
@@ -1182,7 +1182,7 @@ function ShareLink(channel, entity, product_imgurl, product_name, product_brand)
         case 'pinterest':
             var pin_desc = "";
             if (entity == "product")
-                pin_desc = encodeURIComponent((product_brand != "" ? product_brand + ": " : "")) + product_name;
+                pin_desc = (product_brand != "" ? product_brand + ": " : "") + product_name;
```

Comment:
Yes it's already encoded so that's why I removed further encoding. Didn't this solution work?